stacks-signer/src/cli.rs

```diff
@@ -123,6 +132,89 @@ pub struct RunSignerArgs {
     pub config: PathBuf,
 }
 
+#[derive(Parser, Debug, Clone, Copy)]
+/// Arguments for the Vote command
+pub struct GenerateVoteArgs {
+    /// The Stacks private key to use in hexademical format
+    #[arg(short, long, value_parser = parse_private_key)]
+    pub private_key: StacksPrivateKey,
```

Comment:
I think it'd be better to read the key from a config argument.

Signers are already going to have to figure out a secure way to run their signer binary with a configuration file, so this command would just be an extension of that setup.

Reply:
Will do!